### PR TITLE
Make blockTime mandatory, pass it

### PR DESCRIPTION
**Pull request by ArturGajowy**
## Overview
<sup>_What this PR does, and why it's needed_</sup>

Will change base to dev after #2425 is merged

### JIRA ticket:
<sup>_Create it if there isn't one already._</sup>

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: casper/src/test/scala/coop/rchain/casper/helper/BlockGenerator.scala [hunk 1]
```diff
@@ -66,6 +65,25 @@ object BlockGenerator {
...
+      parents <- ProtoUtil.unsafeGetParents[F](b)
+
+      deploys = ProtoUtil.deploys(b).flatMap(_.deploy)
+
+      _ = assert(
+        parents.nonEmpty || (parents.isEmpty && b == genesis),
+        "Received a different genesis block."
+      )
+
+      result <- computeDeploysCheckpoint[F](parents, deploys, dag, runtimeManager, System.currentTimeMillis())
```
Comment (itegulov) [inline casper/src/test/scala/coop/rchain/casper/helper/BlockGenerator.scala]: I unserstand that this is inside tests but should not you use `Time[F]` here among other places?

Comment (ArturGajowy) [reply casper/src/test/scala/coop/rchain/casper/helper/BlockGenerator.scala]: Good point, didn't spot the convention in here :+1:

**Comment by ArturGajowy** [comment]
bors r_

**Pull request merged**